### PR TITLE
feat: examples to connect multiple Nutanix sites and migrate VMs betwe…

### PR DESCRIPTION
**Pull request by codinja1188**
Part of #68

### CODE REVIEW

File: examples/nutanix-clusters/main.tf [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+}
+
+# Configure the Equinix Metal credentials.
+provider "equinix" {
+  auth_token = var.metal_auth_token
+}
+
+module "nutanix_cluster1" {
+  source  = "equinix-labs/metal-nutanix-cluster/equinix"
+  version = "0.1.2"
```
Comment (displague) [inline examples/nutanix-clusters/main.tf:38]: Your changes from #75 are included in "0.2.0"

File: examples/nutanix-clusters/terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1,17 @@
+metal_auth_token       = "iqX9MwH53XrjB7u3nYz15AveD7cqDSU8"     # Equinix Metal API token
```
[the remark this answers is not in the document]
Comment (codinja1188) [reply examples/nutanix-clusters/terraform.tfvars]: updated

**Review by displague — DISMISSED** [review]
Merging to ease review and testing ahead of a release

**Comment by displague** [comment]
I can't merge with the current linting errors. Please correct them (I've opened a PR here: https://github.com/codinja1188/terraform-equinix-metal-nutanix-cluster/pull/1)

You will also need to rebase the branch to get the changes from #82 that this PR depends on to successfully lint.

It may help to install [pre-commit](https://pre-commit.com/) and [trivy](https://aquasecurity.github.io/trivy/v0.53/getting-started/installation/) then run `pre-commit install`, and `pre-commit run --all-files`.

File: examples/nutanix-clusters/main.tf [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+data "equinix_metal_vrf" "nutanix" {
+  count  = var.create_vrf ? 0 : 1
+  vrf_id = var.vrf_id
+}
+
+module "nutanix_cluster1" {
+  source             = "equinix-labs/metal-nutanix-cluster/equinix"
+  version            = "0.3.1"
```
Comment (displague) [inline examples/nutanix-clusters/main.tf:76]:
```suggestion
  version            = "0.4.0"
```

Comment (ctreatma) [reply examples/nutanix-clusters/main.tf]: @codinja1188 The 0.4.0 release was pushed with the wrong tag.  I went ahead and added the v0.4.0 tag and updated the existing release to point to the correct tag.

**Comment by displague** [comment]
We're getting flakes related to hashicorp providers, in a previous CI build it was `local` and now it is `random`.
```
Terraform validate......................................................................Failed
- hook id: terraform_validate
- exit code: 1
Command 'terraform init' successfully done: modules/ssh
Validation failed: modules/ssh
╷
│ Error: registry.terraform.io/hashicorp/random: the cached package for registry.terraform.io/hashicorp/random 3.6.2 (in .terraform/providers) does not match any of the checksums recorded in the dependency lock file
│ 
│ 
╵
Error: Terraform exited with code 1.
```

**Pull request merged**